wd_core.au3

```diff
@@ -1285,13 +1285,12 @@ Func _WD_Startup()
 
 		__WD_ConsoleWrite($sFuncName & ": OS:" & @TAB & @OSVersion & " " & @OSType & " " & @OSBuild & " " & @OSServicePack)
 		__WD_ConsoleWrite($sFuncName & ": AutoIt:" & @TAB & @AutoItVersion)
-		__WD_ConsoleWrite($sFuncName & ": au3WD UDF:" & @TAB & $__WDVERSION & $sUpdate)
+		__WD_ConsoleWrite($sFuncName & ": WebDriver UDF:" & @TAB & $__WDVERSION & $sUpdate)
 		__WD_ConsoleWrite($sFuncName & ": WinHTTP:" & @TAB & $sWinHttpVer)
 		__WD_ConsoleWrite($sFuncName & ": Driver:" & @TAB & $_WD_DRIVER & $sDriverBitness)
 		__WD_ConsoleWrite($sFuncName & ": Params:" & @TAB & $_WD_DRIVER_PARAMS)
 		__WD_ConsoleWrite($sFuncName & ": Port:" & @TAB & $_WD_PORT)
-	Else
-		__WD_ConsoleWrite($sFuncName & ': ' & $sCommand)
+		__WD_ConsoleWrite($sFuncName & ": Command:" & $sCommand)

```

Comment:
The original thought here is that `$_WD_DRIVER` and `$_WD_DRIVER_PARAMS` were already being shown above, so `$sCommand`, which is derived from these two variables, was only shown when $_WD_DEBUG <> $_WD_DEBUG_Info.

Reply:
I revert it back and make speparate bigger PR.

Reply:
As you can see wd_core.au3 is not changed in this PR anymore.